redfish-core/lib/hypervisor_system.hpp

```diff
@@ -436,27 +518,95 @@ inline void parseInterfaceData(
     jsonResponse["DHCPv4"]["DHCPEnabled"] =
         translateDHCPEnabledToBool(ethData.DHCPEnabled, true);
 
+    jsonResponse["DHCPv6"]["OperatingMode"] =
+        translateDHCPEnabledToBool(ethData.DHCPEnabled, false) ? "Stateful"
```

Comment:
Lets keep this in sync with the BMC's DHCPv6. Enable/Disable values from the schema version1.8.0

Reply:
Done